packages/api/portfolioDrafts/funding/createFundingStep.ts

```diff
@@ -100,112 +67,18 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
   }
   return new ApiSuccessResponse<FundingStep>(fundingStep, SuccessStatusCode.CREATED);
 }
+const handler = middy(baseHandler);
+handler
+  .use(xssSanitizer())
+  .use(jsonBodyParser())
+  .use(
+    validator({
+      inputSchema: wrapSchema(schema.FundingStep),
+      ajvOptions: { strict: false },
```

Comment:
Might want to add a `TODO` here as a reminder to change this back to true later, along with a ticket if available.

Reply:
I removed `example: "TaskOrder_9876543210123.pdf"` and was able to get it pass with strict mode.